### ISSUE TITLE
Latest update stops Homebridge working

### ISSUE THREAD

**Issue by robboz4** [oznu/homebridge-config-ui-x#621]
**Describe The Bug:**
Interface told I had 4 plugins to update. First was UI. After install Homebridge never returned. 

**To Reproduce:**
Update Ui code from running system

**Expected behavior:**
Should return to UI after update. It usually does this but not this time

**Logs:**

```
Show the Homebridge / Homebridge Config UI X logs here.

homebridge.service - Node.js HomeKit Server
   Loaded: loaded (/etc/systemd/system/homebridge.service; enabled; vendor preset: enabled)
   Active: activating (auto-restart) (Result: exit-code) since Thu 2020-04-16 18:51:08 PDT; 6s ago
  Process: 3293 ExecStart=/usr/bin/homebridge $HOMEBRIDGE_OPTS (code=exited, status=143)
 Main PID: 3293 (code=exited, status=143)

Apr 16 18:51:08 Homebridge systemd[1]: homebridge.service: Unit entered failed state.
Apr 16 18:51:08 Homebridge systemd[1]: homebridge.service: Failed with result 'exit-code'.

**Homebridge Config:**

```json
Show your homebridge config.json here
```

**Screenshots:**

**Environment:**

* **Node.js Version**: 
* **NPM Version**: 
* **Homebridge Version**: 
* **Homebridge Config UI X Version**:
* **Operating System**: Raspbian / Ubuntu / Debian / Windows / macOS / Docker
* **Process Supervisor**: Docker / Systemd / init.d / pm2 / launchctl / hb-service / other / none

**Comment by oznu** [comment]
Try re-installing Homebridge / the UI.

**Comment by robboz4** [comment]
Tried removing and reinstalling the UI and still get this:
 homebridge.service - Node.js HomeKit Server
   Loaded: loaded (/etc/systemd/system/homebridge.service; enabled; vendor preset: enabled)
   Active: activating (auto-restart) (Result: exit-code) since Fri 2020-04-17 13:54:22 PDT; 748ms ago
  Process: 1566 ExecStart=/usr/bin/homebridge $HOMEBRIDGE_OPTS (code=exited, status=143)
 Main PID: 1566 (code=exited, status=143)

tried older versions of working config.json files. Same issue.
Updated npm same issue..
This set up has been running homebridge without issues for many months. I added the UI which worked perfectly doing updates for plugins and homebridge. But this last update seems to have blown up my stable set up completely.

Any clues as to why?

Will I  have to reinstall homebridge and start all over again?
Cheers
